src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java

```diff
@@ -100,24 +100,16 @@
     @Argument(fullName = "normal_lod", optional = true, doc = "LOD threshold for calling normal variant non-germline.")
     public double NORMAL_LOD_THRESHOLD = 2.2;
 
-    /**
-     * Which annotations to add to the output VCF file. By default the tool adds all of the following annotations.
-     * If an annotation that a filter depends upon is absent, then the particular filtering will not occur and no warning will be given.
-     */
-    @Advanced
-    @Argument(fullName="annotation", shortName="A", doc="One or more specific annotations to apply to variant calls.", optional = true)
-    protected List<String> annotationsToUse = new ArrayList<>(Arrays.asList(new String[]{"Coverage", "DepthPerAlleleBySample",
-            "TandemRepeat", "OxoGReadCounts", "ReadPosition", "BaseQuality", "MappingQuality",
-            "FragmentLength", "StrandArtifact"}));
 
     /**
-     * Which groups of annotations to add to the output VCF file. The single value 'none' removes the default group.
-     * Note that this usage is not recommended because it obscures the specific requirements of individual annotations.
+     * Set of annotation arguments to use.
      * Any requirements that are not met, e.g. failing to provide a pedigree file for a pedigree-based annotation, may cause the run to fail.
-     * For somatic analyses, the StandardSomaticAnnotation group currently contains one annotations: StrandArtifact.
      * Note the latter is redundant to an annotation given by the --annotation argument default.
```

Comment:
This comment no longer makes sense: "Note the latter is redundant to an annotation given by the --annotation argument default"

Reply:
done